src/Magnum/ImGuiIntegration/Context.hpp

```diff
@@ -498,13 +498,28 @@ namespace Implementation {
    the "fallback implementation if there's no other more specialized template"
    behavior. */
 template<class Application, class> struct ApplicationClipboard {
-    static void connect(ImGuiIO&, Context&, Application&) {}
+    static void connect(Context&, Application&) {}
 };
 
 /* Application implementations where returned clipboard contents don't have to
    be freed, such as GlfwApplication */
 template<class Application> struct ApplicationClipboard<Application, typename std::enable_if<std::is_same<decltype(std::declval<Application>().clipboardText()), Containers::StringView>::value>::type> {
-    static void connect(ImGuiIO& io, Context&, Application& application) {
+    static void connect(Context&, Application& application) {
+        #if IMGUI_VERSION_NUM >= 19110
+        ImGuiPlatformIO& platformIO = ImGui::GetPlatformIO();
+        platformIO.Platform_GetClipboardTextFn = [](ImGuiContext*) {
+            void* state = ImGui::GetPlatformIO().Platform_ClipboardUserData;
```

Comment:
Can't you access the PlatformIO from the passed context pointer in some way?

I don't like how implicit the context is here, and looking closer I didn't have it correct originally either. It's called as `_imgui.connectApplicationClipboard(*this)` and one could originally assume that it deals with the context associated with `_imgui`, but it doesn't hold and it instead deals with whatever is "current". So, like, if I have `_imgui` and `_imgui2` in some test app, and call `connectApplicationClipboard()` on both, it'll do it just for one of the two and not both, or potentially some other context entirely, or nothing at all (crash?) if no ImGui context is current.